src/encoded/static/components/award.js

```diff
@@ -2052,6 +2052,8 @@ class CumulativeGraph extends React.Component {
                             ticks: {
                                 autoSkip: false,
                                 maxTicksLimit: 25,
+                                maxRotation: 45,
```

Comment:
@forresttanaka chart.js stopped tilting the numbers probably because of the change in sampling rate. This forces the tilt again.

Reply:
OK, that sounds likely.